app/Util/Helpers.hs

```diff
@@ -3,10 +3,25 @@
     Description : Contains general-use helper functions.
 -}
 module Util.Helpers
-    (safeHead) where
+    (safeHead, returnImageData) where
+
+import qualified Data.ByteString as BS
+import qualified Data.ByteString.Base64 as BEnc
+import Happstack.Server
```

Comment:
Similar to my above comment, modify this import to explicitly list the functions and/or types that are being imported and used in this module